### ISSUE TITLE
Important point missed

### ISSUE THREAD

**Issue by soutchilin** [MicrosoftDocs/powerapps-docs#2067]
Hello Team,

An important point is missed here. To access the D365 CE Org with a AAD token, the right scope need to be set in the request to Azure AD. The scope has to be the URL of the D365 CE organization. A ticket with a default scope (in the Azure App Registration this is "Application ID URI" value) will not work. Please add the info to better enable developer community.

Regards,
Alexander

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 11caa291-2e98-ac3a-d5d1-ecd416b93bdf
* Version Independent ID: 4f30cddd-8849-1887-ada0-a83a3d2a0027
* Content: [Tutorial: Register an app with Azure Active Directory (Microsoft Dataverse) - Power Apps](https://docs.microsoft.com/en-us/powerapps/developer/data-platform/walkthrough-register-app-azure-active-directory)
* Content Source: [powerapps-docs/developer/data-platform/walkthrough-register-app-azure-active-directory.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/developer/data-platform/walkthrough-register-app-azure-active-directory.md)
* Service: **powerapps**
* GitHub Login: @paulliew
* Microsoft Alias: **jdaly**

**Comment by zkirby** [comment]
I cannot stress how important it is we get this added to the docs - I spent a solid 2 days debugging, flipping, toggling, and toying with this entire process trying to get it to work. I was on my hands and knees begging to every notable deity to no avail. I stumbled onto this comment out of sheer desperation, clicking every link I could find on the docs.

I've set up CRM integrations for a variety of vendors - and this was by far the most cumbersome and complex. I know the innate complexity of the msoft ecosystem makes simplifying this process a daunting task; but, I find it incredibly frustrating that if this had been included in the docs from the beginning I would have been saved many hours of pain.
